### DISCUSSION TITLE
Hallo zusammen ... zwei  Wechselrichtungszähler Strom ... Hello everyone ... two alternating direction meters electricity

### DISCUSSION THREAD

**Discussion by hardy196** [jomjol/AI-on-the-edge-device#2315] [Q&A]
Hallo ich habe einen Stromzähler der in zwei Richtungen zählt ... 1.8.0 Verbrauch Stadtwerke / 2.8.0 was eingespeist wurde . PV-Anlage.  Kann man beide Werte ermitteln ?   oder ist es möglich einen der Wert zu erfassen ?

Hello, I have an electricity meter that counts in two directions... 1.8.0 consumption by public utilities / 2.8.0 what was fed in. PV system Can you determine both values? or is it possible to capture one of the values?

![IMG_5709](https://user-images.githubusercontent.com/96956341/232247299-ac9a0e69-87a7-48b3-bafa-05b034b6edb2.jpg)

![IMG_5708](https://user-images.githubusercontent.com/96956341/232247304-a4e40c69-9b9d-4d04-bb9f-e5ae35570a11.jpg)

**Comment by flyingToast3r** [comment]
I can't answer your question, hopefully others can, but I just want to hint that by using the IR-interface of your reader (like project volkszaehler) you would get rid of this problem and have a much more reliable data source.

**Reply by caco3** [reply]
Dazu müsste der Zähler die verschiedenen Anzeige-Modis unterschieden können.
Das wird von vielen gewünscht, ist momentan aber (noch) nicht unterstützt.
Siehe auch https://github.com/jomjol/AI-on-the-edge-device/issues/1936

**Comment by rainerhgit** [comment]
Ich kann auch nur empfehlen, den Zähler mit einem IR-Sensor auszulesen. Dabei sind alle Daten verfügbar (Gesamtwerte, Aktuelle Werte ). Bei meiner Konfiguration z.B. werden die Daten im ioBroker mit dem "Smartmeter" Adapter ausgelesen. Funktioniert perfekt.
(Und mein PV-Wechselrichter liefert ihre Daten auch in ioBroker)

**Comment by friedpa** [comment]
Same in my setup, works very good as descript bei @rainerhgit. For Austrians: have a look at: www.shrdzm.com
